### DISCUSSION TITLE
虚数神女录【米哈游宇宙二创】

### DISCUSSION THREAD

**Discussion by Sqrootof** [share121/inter-knot#2148] [其他]
**第一章：那年有个少女**
　　空在一个幽静的暗室中醒来，身边放着一柄生锈的剑。
　　石壁之上镶嵌着青铜古灯，壁上绘画繁复，彩绘的壁画保存完好，栩栩如生，没有丝毫的剥落。
　　一袭白衣古静如素，那张年轻的少年脸庞在昏暗的石室间清秀如同少女。
　　他看着那柄锈迹斑斑，毫无灵气的古朴长剑，默然许久，他终于幽幽叹了一口气：「临渊羡鱼，终于被深渊吞噬了。」
　　他推开石门，走进了光里。这一日，这个尘封了五百年的府邸终于洞开。微风扑面，有些涩，有些冷。
　　万水依山渐入心怀，五百年一场大梦，他恍然初醒，默默领会着这五百年闭关的感悟。
　　山峰很高，高耸入云，耳畔可闻鸟语，也可以听到飞瀑溪流漱雪碎玉般穿过云雾的声音。
　　少年看着石壁间飞泄而出的溪水，看着白云深深，不知何处。若有所思。
　　他看着自己的双手，无奈地笑了笑：「修道五百年，尽付水云间？」
　　说完这句话，他开始不停咳嗦，咳嗦声在寂静的山谷中显得格格不入。
　　咳嗦许久之后，他终于抬头望向云层掩映之间的青山，那是潮断峰的母峰，相比子峰更为巍峨高耸，孤绝苍翠。他的目光有些狂热，有些茫然，有些不甘，最后竟然有些害怕。
　　五百年前，他便是令使境巅峰。终于偶得机缘，有望达到世人从未到达的境界。便在潮断峰闭了一个五百二十年的大关。如今他提前出关。却发现自己令使境界如海的法力都消失得一干二净。但是自己的境界却大涨，隐隐快要跨过那个门槛。如今自己的容颜青稚如同少年便是最好的证明。淬体炼魄，拔污除秽之后，他这副身躯便返璞归真至了少年。
　　但是空有境界没有法力施展，和废人有什么区别？他需要二十年时间来解决自身出现的问题。
　　他缓缓走下山崖。山崖依旧，无论是石道还是风景都如同五百年前一样。只是尘世不比山水，人间可不只是千篇一律的山水更替，世俗人伦沧海桑田，不知道已经到了哪一步。
　　随着他拾级而下，他竟然能够明显感受到自己空空荡荡的剑胎之内，缓缓流入灵气。仿佛是溪流缓缓地流入干涸开裂的海床，虽然杯水车薪，但是百川东到海，总有充盈的那一日。他放慢了脚步，开始推演。
　　总有人把人间比作棋盘。只是人间的事远远比下棋复杂太多，即使是最精通算计推理的人也只能算出一个大概罢了。
　　他的脚步越来越慢，直至停下。
　　出了潮断峰子峰自己设立的禁制的范围之后，他听到了有人说话的声音。他的推演被说话声硬生生打断，这让他有些烦躁。
　　不远处隐隐约约站着一个女子，隔着树林花影，那女子一身黑白的单衣犹显古意，仿佛山水之间一道难以捉摸的窈窕写意。空身躯微震，他觉得这个身影好生眼熟。正在他思考之际，一个男子说话的声音打断了他的思绪。
　　「都过去五百年了，刻仙子还如此念念不忘么？这些日子我结庐山下，时常看到刻仙子御剑山灵，在潮断峰外徘徊的流光魅影。甚是仰慕。」
　　空这才注意到年轻女子对面站着一个身材干瘦穿着黑白道衣的中年男子。
　　刻仙子……她是……空的嘴角无声扬起，没想到命运如此巧合，自己刚刚出关便见到了自己五百年前最寄予厚望的首徒，刻晴。
　　只听刻晴极其冰冷道：「我剑宗行事，关你阴阳阁何事？」
　　那人冷笑道：「刻仙子不愧是璃月王朝女剑仙魁首，如今敢负剑行走天下的女子，早就屈指可数了。」
　　刻晴只是说道：「希望二十年后你还能如此说话。」
　　那人放声狂笑：「二十年？你以为那个人真的能出关么？别傻了，如今全天下都知道，那……」
　　话未说完，一道剑光照彻了青山。仅仅是一瞬间，刻晴的剑尖便顶在了那人的喉咙口。
　　她平静道：「再让我听到你诬蔑家师，我就杀了你。」
　　那人竟然丝毫不为所动。淡然道：「刻晴啊刻晴，虽然我境界远不如你，但是别人不知道，我可知道，你如今不过……」
　　忽然，那阴阳阁的道人神色一厉，转头望向林间，目光如炬如电：「谁在那里。」
　　空微微一震，他刚刚出关，还没能熟练运用道法隐匿气息，竟然被发现了。
　　刻晴的目光也望向了这里，无奈之下，空只好缓缓走出林间，看着眼前两人，他想了想，弯腰作揖：「见过两位仙长。」
　　刻晴看着已经抬起头的他，微微蹙眉，问道：「你是哪个仙门的弟子？」
　　空看着这位曾经的徒弟，她已然那么美丽，清丽的容颜，高高盘起的秀发，斜插的木簪，一丝不苟的黑白剑装裹着她傲然挺拔的身材，仿佛她就是一柄矗立林间的剑，所有的山水景色都被夺去了锐气。他感到很欣慰，自己这位首徒不仅出落得更加娉婷，也迈过了那一道剑道门槛。只可惜，此刻自己无法与之相认。
　　空看着刻晴，平静道：「我没有宗府门派。我是璃月王朝的一个庶子。我叫哲。」
　　五百年前，自己为了防止各种不测，早已埋下了许多补救的方法，这个身份在五百年前便已设计好了。从此，那个叱咤风云的空便死了，活着的是名为哲的白衣少年。
　　刻晴看着他，忽然说道：「你愿意随我修行，追求剑道么？」
　　哲心中一惊，心想自己的首徒收徒弟就这么随便么？这时，那个阴阳阁的中年人发出了一串尖锐的笑声：「没想到堂堂刻仙子如今已经如此……如此饥不择食了？哈哈哈，你们剑宗已经实在招不到人了么？这种路边随意见了一面的人都要？」
　　刻晴没有理会他的冷嘲热讽，又问了一遍：「你愿意么？」
　　那人咧了咧嘴，忽然开口道：「这位公子，你别急着答应。我是阴阳阁的四长老斯科特。虽然实力不算拔群，但是在阴阳阁地位也算非凡，这位公子可愿随我去阴阳阁修行？」
　　刻晴神色一厉，目光如剑。那位自称斯科特的长老笑道：「怎么，刻仙子不高兴了，我斯科特就是要和你抢人。」
　　斯科特继续说道：「我阴阳阁在璃月皇朝的地位你不会不知道吧？如今这位刻仙子的宗门早已中落，独木难支，不管你天赋高低，根骨好坏，进入剑宗是一个极差的选择。」
　　哲很想告诉他，他真的不知道。
　　刻晴冷冷道：「斯科特，你当真以为我不敢杀你。」
　　斯科特伸长了脖子一阵冷笑，一副你来啊的样子。在他心中，璃月王朝没有任何年轻人可以拒绝成为阴阳阁弟子的诱惑，而且这种空有皮囊的庶子对力量最为渴求，如今他没有马上答应下来估计只是想给这位璃月皇朝女子剑道魁首一点面子罢了。不管这个人资质怎么样，总之不能让刻晴收走，自己就是摆明了打压她。
　　刻晴收剑而立，看着哲，她自己也没了信心，只是发出了一声弱不可闻的叹息。
　　正当她想要驭剑离开之际，哲忽然看着她，缓缓开口道：「我跟你走。」
　　刻晴娇躯一震，匪夷所思地看着他。
　　斯科特更是瞪大了眼睛，用一种看疯子和白痴的眼神看着他，仿佛这是世界上最匪夷所思的事情。他气得面容都有些扭曲，竟是不自禁笑了出来，「你知道你错过了什么吗？」
　　他又冷笑道：「真是初生拧，剑宗注定是死路，今天如此，二十年后也会如此，大道机缘你不走，你自己要找死我也不拦着你了。下次见面我要亲手剐了你！」
　　哲没有理会他，他缓缓走到刻晴身边，此刻他少年身材的身高只能到刻晴的肩膀，曾经经常被自己宠溺揉头的少女此刻居然比自己要高了，他忽然觉得好不自在。
　　他看着刻晴，说道：「带我去剑宗吧。」
　　…………
　　寒宫剑宗位于璃月皇朝的南端，建于归雪峰上，临近月海。
　　这个世界名为虚数界，大致分为四个势力，人间的大陆王朝版块，璃月皇朝。南方九万里月海绕城而过的仙舟城，那是白发族的住所。一直被几大妖族割据混战，不得安宁的北域。还有凌驾与人间之上，聚集了最多九境以上飞升者的天空岛。而寒宫剑宗是刻晴一手建立的，是璃月皇朝的六大宗门之一。
　　刻晴带着哲驭剑赶路的时候没有说太多话，只是和他交待了一些大致的宗门内容和需注意的事宜。简单而琐碎。
　　寒宫剑宗很大，但是入宗却只能感受到凄清。
　　一路驭剑而来寒风蚀骨，虽然刻晴已经给他加持了许多保护，但是如今羸弱的身躯仍然侵入了许多风寒。他忽然想起来自己当年也是这么带着她驭剑的，只是他当时可没有刻晴这样细致，一路驭剑下来把她冻了个半死，小姑娘还格外倔强，一路上一声不吭。
　　想起这些陈年旧事，他不由地微微扬起嘴角。
　　一道剑光落在寒宫之前，清冽惊艳。刻晴收剑入鞘。哲仰头，目光缓缓向上，一直落到那两个寒玉雕琢的青蓝色大字上：
　　寒剑。
　　寒宫清幽照人。刻晴领着他走入殿口。殿门上空剑气纵横，寒光闪耀，若是初出茅庐的人见到如此凛冽剑气，必然会心驰神遥。但是哲却平静得出奇。
　　这位堪称璃月王朝剑道魁首的绝美少女望着哲，缓缓开口道：「我不知道你为什么要随我修剑。或许是钟情于剑，或许只是一时冲动，考虑不周。但是不管是因为什么，只要随我踏进了这扇门，你从此便是我的弟子。你的生命便与剑息息相关，连为一体。你愿意么？」
　　哲静静地看着她，竟是有些犹豫。
　　刻晴微微叹息，拍了拍他的肩膀：「让你这么快做这么仓促的决定确实太为难你了，这是我的错，不怪你，如果你现在反悔，我可以护送你下山。」
　　哲摇头道：「不是因为这个。」
　　刻晴纤长的秀眉微蹙，等待着他继续说下去。
　　哲没由来地挠了挠头，竟然有些支支吾吾道：「我愿意追求剑道，只是……我能不叫你师父么？」
　　曾经缠着自己一声声叫师父的女孩，如今自己反过来要叫她师父，他还是很难适应。
　　刻晴疑惑道：「为什么？」
　　哲很快编了一个借口：「我曾经有一位师父，教我读书写字，年前他病逝了。我很敬重我的师父，短时间内我不想找其他师父。」
　　刻晴看着他的眼角，两双清澈好看的眼神对视着，她似乎是在辨认哲是否说谎了。片刻之后，她才幽幽道：「节哀。」
　　说着，她转过身牵起了哲的袖子走入寒宫之中。哲抬起脚跨过了那道不算高的门槛。
　　一步跨过，剑道九境。他便水到渠成般来到了第一境。
　　第一境对于大部分普通人来说，穷尽一生都无法跨过。这是天地堑。但是在此刻的他眼中，不过一道矮矮的门槛。
　　刻晴没有察觉到他的变化。
　　入了寒玉殿，一对穿着素衣剑袍，英气逼人的少年少女走到刻晴面前，鞠躬作揖：「见过师父。」
　　这是这偌大的寒宫剑宗仅剩的两名弟子了。
　　刻晴简单介绍道：「他叫行秋，是你的二师兄。她叫香菱，是你的大师姐。你是寒宫第三位弟子。」
　　哲沉默了一会，说道：「我还没有认你做师父。」
　　名为行秋的少年正欲开口，那眉清目秀的少女香菱便怒气冲冲道：「怎么？你看不起我们剑宗啊！你也想去修那些邪魔外道？那你别来啊，外面前途一片光明。」
　　哲看着这位鼓着香腮怒气冲冲的少女，感觉很像当年的刻晴，他本就不爱说话，所以一时间更不知道说什么好。
　　刻晴打破尴尬，柔声道：「他叫哲。不叫我师父是另有隐情，并非对剑宗有何异端看法。以后你们好生相处，莫要欺负他。对了，玄言，等会你随我入正殿，我给你讲一下入门心法。」
　　谈话间，一道素白色的茸片从灰蒙蒙的天空上悠悠飘落。
　　秋风散尽，林木苍黄。那是初冬的第一片雪。
　　香菱笑着摊开了手掌，咬着嘴唇接下了这一瓣雪花，那一瓣雪花转瞬消逝，但是她仍然欢天喜地道：「下雪啦下雪啦！」
　　越来越多的雪花从铅灰色的云层中坠落，簌簌飘零向层峦青山之间。行秋看着满天雪花，也喃喃道：「冬至了。」
　　刻晴和哲望着悠悠扬扬的漫天飞雪，似是都思及了什么往事，都沉默不言。
　　那年冬……哲忽然笑了，他摊开手掌。雪花落在掌间。他合上十指握住了这片雪。
　　这一刻，他迈入了剑道第二境……
　　「剑道和其他道一样，都分为九重境界，每三重境界都是一个槛。达到七境以上便可以进入那座高高在上的天空岛。而九境之上是原神。我此刻的境界便是原神。」原神是真正的大宗师境界，无论放在哪里都是可以开宗立派的至高存在。但是刻晴说这话的时候却极其平静，那不是故作谦虚，而是真正的平静。
　　「原神之上是令使。」说道这里她顿了顿。她补了一句：「我师父便是令使巅峰的剑修。」
　　说这句话的时候她还是很平静。但是哲看得出来，她是在故作平静。哲故意问道：「请问你师父现在身在何方？」
　　刻晴用一种很奇怪的眼神看着他。她的语气中带着一丝微微的骄傲：「全天下都知道我师父是空。五百年前纵横整个大陆最天才的剑修。五百年前，师父得到了大道机缘，于潮断峰闭关。我在潮断峰见到你，还以为你知道我师父的事情。」
　　哲摇了摇头，说道：「我出生陋僻。所以不得而知。」
　　刻晴只是说道：「师父是我最敬重的人。」
　　哲觉得又有趣又可爱。他很想告诉她，自己就是你最敬重的师父大人空。然后像以前那样宠溺地揉她的脑袋。但是出于诸多考虑，他微动的手指还是缩了回去。
　　刻晴看着哲说道：「现在我传你寒宫入门剑法心得。你一下要记下来。」
　　「嗯。好。」
　　刻晴继续道：「记口诀很容易。但是想要真的迈过那道槛，真正登堂入室却是极难，如果三个月时间你无法进入一境，那便基本与剑道无缘。到时候你来去都由自己决定。」
　　哲点了点头，只说了一个好字。
　　出了寒玉宫，香菱走到上来，上下打量了他一番，说道：「小师弟啊，你长得挺好看的。」
　　哲倒是没有反驳小师弟这个说法，五百年前他听过太多太多夸奖，如今被一个初出茅庐的少女夸奖，只是觉得有些新奇。
　　香菱戳了戳他，有些不满道：「你大师姐和你说话呢。你居然敢不理？」
　　哲只好说道：「我知道我很好看。」
　　他不喜欢说话，所以也不太会和人打交道。
　　香菱瞪大了眼睛，她的眼睛墨色很深却很干净，像是砚好的新墨一样，让人忍不住看了又看。「哇，小师弟，没想到你这么自恋。」
　　「……」
　　香菱拍了拍自己初初长成的胸脯，说道：「你有什么问题都可以问姐姐我，如果你在山下被人欺负了，师姐可以替你报仇的。」
　　哲确实有很多问题，比如他最想问的，为什么五百年前最为辉煌的剑道如今没落至此？
　　但是他终于没有开口。只是说：「谢谢师姐。」
　　那些问题虽然是很大的问题，但是对于此刻剑心已经修到半步律者的他来说，都不重要了。无论五百年间发生了什么天翻地覆的变化，只要隐忍二十年，他便能复兴剑道。只是……
　　看着满天纷纷扬扬的落雪，他忽然想念自己的未婚妻了。
　　天空岛神王宫的圣女甘雨。
　　五百年了，你还好么？
　　…………
　　宫殿口的雪越落越高。
　　白茫茫地遮住了远山近树，一点点堆砌在本就雪白的砖瓦上。远远望去犹似一座清寒蟾宫。天地间唯一的颜色里，刻晴披着白色绒边红色面料的披风站在风雪之中，她没有用法力隔绝雪花，任由它们落在自己刀削般的香肩上，沾濡在青黑的秀发上。像是瀑布上的小花，也像是星空下的梅瓣。
　　一道黑白色的剑光在她身边绽放，寒宫之中闪起了千万道剑光，那些黑白分明的剑光仿佛是她衣襟上飘起的裙带，也像是她眸子俯瞰世界的样子。
　　洋洋洒洒的雪花也被黑白两色照亮。
　　哲站在殿前，忽然回身凝望，漫天的剑光照亮了他的眸子，如果是过去的话，他会觉得这些剑光太单薄，运气剑气的方式太过简单，挥剑的速度也不够凌厉。
　　但是此刻他只是觉得很美。就像那位挥舞剑气的少女一般。
　　行秋不知何时出现在了他的身后，拍了拍他的肩膀，说道：「很好看吧？」
　　哲平静地看着他：「很好看。」
　　行秋缓缓开口：「我不知道你到底是谁，是不是某个宗门派来的卧底，但是如果你敢加害师父，我一定会亲手杀了你。」
　　哲没有理会他，缓缓离开了正殿。
　　接下来的一个月过得无比平静而简单，他早已不需要练剑了。他练过太多太多的剑，从前一天挥剑何止百万次？每一个轨迹和行气方式都早已烂熟于心。对他来说，练剑还不如发呆更有意义。这一个月刻晴都悉心教导他们剑法，行秋的悟性很高，学剑很快。香菱也不算逊色，只是这个小姑娘有些静不下来。哲一直表现得不温不火，他挥剑挥得很好看，但是一直被香菱嘲笑是花架子。
　　但是这一天，刻晴没有教他们练剑，寒宫的雪还没有停，天地间依旧覆着浅浅颜色。
　　哲将那本自己年轻时候编著的《剑气初行之理》随意摊在桌上，这本书写得很简单，但是内容很不简单。但是不管简单不简单，他都不想看。
　　因为书上的每一个字，甚至笔画的高低他都记得。
　　百无聊赖之后，他推开了小小的厢房，凭着感觉在寒宫之间踱步。

**Comment by TAKO-COLO** [comment]
再写50丁尼的！OwO

**Comment by Sqrootof** [comment]
这半片只是替换了名词，如果有人看的话我后续对原文进行一下大修写写
